File: tools/tiktok-farm-window/paths.py
"""Shared paths for tiktok-farm-window."""

from __future__ import annotations

from pathlib import Path

TOOL_ROOT = Path(__file__).resolve().parent
CREATOR_ROOT = TOOL_ROOT.parent / "tiktok-account-creator"
PROFILES_DIR = TOOL_ROOT / ".profiles"
ACCOUNTS_FILE = TOOL_ROOT / "accounts.json"
SAMPLE_VIDEO = TOOL_ROOT / "assets" / "sample.mp4"
SETTINGS_PATH = CREATOR_ROOT / "settings.json"
USERS_PATH = CREATOR_ROOT / "users.txt"

File: tools/tiktok-farm-ts/src/paths.ts
import path from "node:path";
import { fileURLToPath } from "node:url";

const __dirname = path.dirname(fileURLToPath(import.meta.url));

export const TOOL_ROOT = path.resolve(__dirname, "..");
export const REPO_ROOT = path.resolve(TOOL_ROOT, "../..");
export const PROFILES_DIR = path.join(TOOL_ROOT, ".profiles");
export const ACCOUNTS_FILE = path.join(TOOL_ROOT, "accounts.json");
export const SAMPLE_VIDEO = path.join(TOOL_ROOT, "assets", "sample.mp4");
export const SETTINGS_FILE = path.join(TOOL_ROOT, "settings.json");
export const GMAIL_CODE_BIN = path.join(REPO_ROOT, "bin", "gmail-tiktok-code");
